AgentDiscoveries-Backend/src/main/java/org/softwire/training/models/ReportBase.java

```diff
@@ -11,6 +11,17 @@
     private LocalDateTime reportTime; // Always UTC in the DB
     private String reportBody;
     private int agentId;
+    private String reportTitle;
```

Comment:
This is still in the report base file, so it still feels like this is expected to be applied to both types of report?